src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -18,38 +20,55 @@ namespace Microsoft.PowerShell.Commands
     [Cmdlet(VerbsCommon.Format, "Hex", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526919")]
     [OutputType(typeof(Microsoft.PowerShell.Commands.ByteCollection))]
     [Alias("fhx")]
-    public sealed class FormatHex : PSCmdlet
+    public sealed class FormatHexCommand : PSCmdlet
```

Comment:
Agree it's inconsistent, but since this class is public, we shouldn't rename it.

Reply:
@vexx32 Your PR contains many style and unrelated fixes so I'd prefer to merge my #8191 at first.

Reply:
@iSazonov go for it!

It'll be a bit easier for me to do a thorough diff between my changes and yours once #8191 is merged. I don't mind figuring out the merge after yours is done. :smile: